.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+# TODO: do we need this?
+# permissions:
+#   contents: read # for actions/checkout to fetch code
+
+# TODO: verify-codegen, coverage
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version:
+          - "1.19"
+          - "1.20"
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v4
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Build
+        run: make manager
```

Comment:
Why do we need to build manager here?

Reply:
`go test` only builds packages that have unit tests, iirc.

Reply:
Looks like I'm wrong. From `go help test`:
> 'Go test' recompiles each package along with any files with names matching
the file pattern "*_test.go".